.github/workflows/tests.yml

```diff
@@ -662,7 +662,9 @@ jobs:
       matrix:
         app:
           - ping
-          - dhcp_client
+          # Componolit/Recordflux#926: Temporarily disable this test as it is
+          # too slow for the CI.
+          #- dhcp_client
```

Comment:
The usually used scheme is `# ISSUE: Componolit/RecordFlux#926`. This simplifies "grepping" for all references to issues.

Reply:
fixed